docs/project-development-environment.md

```diff
@@ -55,7 +55,9 @@ To run the application tests only, see [Commonly used commands while developing
 
 ## Access the application
 
-[http://localhost:8000](http://localhost:8000)
+`http://<your_ip>:8000`
+
+where `<your_ip>` is your (public) IP.
```

Comment:
On Linux,
```bash
ip route get 1 | awk '{print $NF;exit}'
```
seems to be a pretty reliable way to determine the IP to use. (Found [on StackOverflow](http://stackoverflow.com/questions/13322485/how-to-i-get-the-primary-ip-address-of-the-local-machine-on-linux-and-os-x/25851186#25851186).)

If it answers `RTNETLINK answers: Network is unreachable` one will have to manually add an entry in `/etc/hosts` and use that.

Should any of this be mentioned here in the docu?